pinot-tools/src/main/resources/conf/pinot-tools-log4j2.xml

```diff
@@ -21,11 +21,7 @@
 -->
 <Configuration>
   <Appenders>
-    <Console name="console" target="SYSTEM_OUT">
-      <PatternLayout>
-        <pattern>%d{yyyy/MM/dd HH:mm:ss.SSS} %p [%c{1}] [%t] %m%n</pattern>
-      </PatternLayout>
-    </Console>
+    <Console name="console" target="SYSTEM_OUT" />
```

Comment:
this change seems not related to the problem, right?

Reply:
correct. It's just a cleanup to make the behavior same as before (when we used log4j).